src/main/java/org/osc/sdk/controller/api/SdnRedirectionApi.java

```diff
@@ -74,7 +75,7 @@
     /**
      * Creates an inspection hook on SDN controller, see {@link InspectionHookElement}
      *
-     * @param networkElem  provides network element containing protected ports
+     * @param networkElements  provides a list of network elements containing protected ports
```

Comment:
Add in comments that Inspectionport argument will have the for NSC and Nuage : ingress and egress redirection ports and for SFC it will have the hooked(i.e sfc port chain id) as redirection target.